### PR TITLE
Bump all Python dependencies

### PR DESCRIPTION
**Pull request by mozbhearsum**
Pyup is broken, doing this by hand.

### CODE REVIEW

**Comment by mozbhearsum** [comment]
Test failures are just for the backend, and appear to be caused by a connexion upgrade.

**Comment by mozbhearsum** [comment]
The `connexion` bump ended up breaking a bunch of tests. Most of them were resolved in https://github.com/mozilla-releng/balrog/pull/1126/commits/5d4604a1457eaf42a5e5787fac5629d83c0ab666 and https://github.com/mozilla-releng/balrog/pull/1126/commits/4e45e04bde93f702e7b897a7672ce26e720cb916, which have comments with details about the issues and fixes.

File: .taskcluster.yml [hunk 1]
```diff
@@ -85,7 +85,7 @@ tasks:
                   git clone ${repo_url} balrog &&
                   cd balrog &&
                   git checkout ${head_sha} &&
-                  pip install coveralls==1.9.2 tox &&
+                  pip install coveralls tox &&
```
Comment (rail) [inline .taskcluster.yml]: Shall we move coveralls to `test.in`? It doesn't belong there logically (you don't necessarily submit to coveralls as a   part of every test run), but at least it's pinned.

Comment (mozbhearsum) [reply .taskcluster.yml]: Yeah, that's probably a good idea. I'll try to do it here.

**Comment by mozbhearsum** [comment]
@rail - It's a bit ugly to do this, because we have to pass through a lot of information to tox to do it, but it works :).

**Pull request merged**